lemarche/templates/siaes/_card_tender.html

```diff
@@ -12,6 +12,12 @@
             </div>
             <div class="col">
                 <h2 class="h4 mb-0">{{ siae.name_display }}</h2>
+                {% if  siae.super_badge %}
```

Comment:
```suggestion
                {% if siae.super_badge %}
```
same dans les autres fichiers modifiés ;)